apps/integration-e2e/features/access-keys.feature

```diff
@@ -174,3 +174,12 @@ Feature: Access Keys
       Examples:
         | keys        |
         | p           |
+
+  Scenario Outline: Open recognized text content
```

Comment:
The Scenario has a different indentation than the 2 scenarios above it. Can you fix indentation for the 2 above scenarios? The one you have written is correct. :+1: